torch/utils/cpp_extension.py

```diff
@@ -303,13 +351,20 @@ def unix_wrap_single_compile(obj, src, ext, cc_args, extra_postargs, pp_opts):
             try:
                 original_compiler = self.compiler.compiler_so
                 if _is_cuda_file(src):
-                    nvcc = _join_cuda_home('bin', 'nvcc')
+                    nvcc = (_join_rocm_home('bin', 'hipcc') if ROCM_HOME else _join_cuda_home('bin', 'nvcc'))
```

Comment:
Is there something else we can use to decide if we should be using to build with ROCM or not? Alternatively, creating a helper variable `is_rocm_extension` helps readability

Reply:
Done